datasets/ohsumed/README.md

```diff
@@ -0,0 +1,205 @@
+---
+annotations_creators:
+- human-annotated
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-nc-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 100k< n<500K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- multi-label-classification
+
+---
+
+# Dataset Card for ohsumed
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage: http://davis.wpi.edu/xmdv/datasets/ohsumed.html**
+- **Repository: https://trec.nist.gov/data/filtering/t9.filtering.tar.gz**
+- **Paper: https://link.springer.com/chapter/10.1007/978-1-4471-2099-5_20**
+- **Leaderboard:**
+- **Point of Contact: [William Hersh](mailto:hersh@OHSU.EDU) [Aakash Gupta](mailto:aakashg80@gmail.com)**
+
+### Dataset Summary
+
+The OHSUMED test collection is a set of 348,566 references from
+MEDLINE, the on-line medical information database, consisting of
+titles and/or abstracts from 270 medical journals over a five-year
+period (1987-1991). The available fields are title, abstract, MeSH
+indexing terms, author, source, and publication type. The National
+Library of Medicine has agreed to make the MEDLINE references in the
+test database available for experimentation, restricted to the
+following conditions:
+
+1. The data will not be used in any non-experimental clinical,
+library, or other setting.
+2.  Any human users of the data will explicitly be told that the data
+is incomplete and out-of-date.
+
+Please check this [readme](https://trec.nist.gov/data/filtering/README.t9.filtering) for more details
+
+
+### Supported Tasks and Leaderboards
+
+[Text Classification](https://paperswithcode.com/sota/text-classification-on-ohsumed)
+
+### Languages
+
+The text is primarily in English. The BCP 47 code is `en`
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```

Comment:
It would be nice to have an actual sample here, in JSON format 😄 
Something like: 
```json
{
    "seq_id": <seq_id>,
    "medline_ui": <medline_ui>,
    "mesh_terms": "terms",
    "title": "Title",
    "publication_type": "pub_type",
    "abstract": "Abstract of the document",
    "author": "John Doe",
    "source": "source of the document",
}
```